buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -625,6 +631,10 @@ class BuildPlugin implements Plugin<Project> {
                         jarTask.manifest.attributes('Change': shortHash)
                     }
                 }
+                // Force manifest entries that change by nature to a constant to be able to compare builds more effectively
+                if (System.properties.getProperty("build.compare_friendly", "false") == "true") {
+                    jarTask.manifest.getAttributes().clear()
```

Comment:
Can you elaborate more on this comment? I am concerned debug ability of a build that lacks any of the manifest info we inject.

Reply:
This property should never be passed directly. It's being used by the build compare plugin.
Unfortunately this useful plugin is still missing a lot of features, like understanding that fields in manifest files will change between builds, so with this property we prevent this to reduce noise in build compare. I'm not happy about what this property does either, it will backfire, but it does require one to pull the trigger.